iothub/device/src/ClientPropertyCollection.cs

```diff
@@ -48,17 +47,17 @@ public void AddComponentProperty(string componentName, string propertyName, obje
 
         /// <inheritdoc path="/remarks" cref="AddRootProperty(string, object)" />
         /// <inheritdoc path="/seealso" cref="AddInternal(IDictionary{string, object}, string, bool)" />
-        /// <inheritdoc path="/exception['ArgumentException']" cref="AddRootProperty(string, object)" />
```

Comment:
Xml doc generation tool isn't able to resolve the query string within the inheritdoc path, so we cannot use this pattern.